rclcpp/include/rclcpp/executors.hpp

```diff
@@ -81,7 +95,18 @@ spin_node_until_future_complete(
   return FutureReturnCode::INTERRUPTED;
 }
 
-} // namespace executors
-} // namespace rclcpp
+}  // namespace executors
+
+template<typename FutureT, typename TimeT = std::milli>
```

Comment:
If I read the diff correctly it was always in the rclcpp namespace. I think I organized it like this because this function uses the above function which is in the executors namespace.